synonymdev/bitkit-react-native
# fix(transfer): respect node capacity limit in custom setup

### Description

- Include existing channels when calculating client and lsp balances for the custom setup.
- ChannelList buttons navigate to custom setup (as per design)

### Linked Issues/Tasks

Closes https://github.com/synonymdev/bitkit/issues/1563

### Type of change


### Tests


### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/8562a134-18ab-42b0-8013-eded3332025c

### QA Notes

Custom Setup should not let you open a channel that puts you over the LSP node capacity limit (preventing the channel from being opened)
